Makefile

```diff
@@ -1,12 +1,21 @@
+TEST_COUNT ?= 10
+COVERAGE_FILE ?= coverage.out
 
-ensure_go_1_21:
+build: ensure_go_version
+	go build -v ./...
+
+generate: ensure_go_version
```

Comment:
Recommend we install the correct mockery version (v2.43.0 seems like)

Reply:
Responding above.